code/datums/keybinding/human.dm

```diff
@@ -9,9 +9,9 @@
     full_name = "Quick equip"
     description = ""
 
+
 /datum/keybinding/human/quick_equip/down(client/user)
-    var/mob/living/carbon/human/H = user.mob
-    H.quick_equip()
+    SEND_SIGNAL(user.mob, COMSIG_KB_QUICKEQUIP)
```

Comment:
```suggestion
    if(SEND_SIGNAL(user.mob, COMSIG_KB_QUICKEQUIP) & comsig keybind handled or something) 
          Return true
```
Do the same for the others, only return on keybinds if it actually does the bind or you can't double up this key for xenos who don't need it